lib/features/support/env.rb

```diff
@@ -57,7 +57,11 @@ def write_failed_requests_to_disk(scenario)
         end
         file.puts
         file.puts "BODY:"
-        file.puts JSON.pretty_generate(request[:body])
+        if request[:request].header["content-type"].first == 'application/json'
+          file.puts JSON.pretty_generate(request[:body])
+        else
+          file.puts "The request body was a non-JSON format"
```

Comment:
Is it possible to pass `request[:body]` to `file.puts`? If so, this would be quite helpful for debugging scenarios failures where the request isn't JSON, such as in the bugsnag gradle plugin